test/e2e/vmagent_test.go

```diff
@@ -47,8 +47,9 @@ const (
 )
 
 var (
-	ICMPType = int32(8)
-	ICMPCode = int32(0)
+	ICMPRequestType  = int32(8)
+	ICMP6RequestType = int32(128)
+	ICMPRequestCode  = int32(0)
 )
```

Comment:
would you move these? it doesn't make much sense to have them defined in this file, especially if we start using them for other e2e tests

maybe we could add a `net_constants.go` file? there are a few constants / variables in other files (e.g. `protocolICMP` in `traceflow_test.go`) that could be moved to this new file as well. What do you think?